src/atoms/TextInput/Label.native.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Platform } from 'react-native';
+import styled from 'styled-components/native';
+import Text from '../Text';
+import View from '../View';
+import Space from '../Space';
+
+const IS_ANDROID = Platform.OS === 'android';
+
+const PADDING_TOP_MULTIPLIER_ANDROID = 0.36;
+const PADDING_TOP_MULTIPLIER_IOS = 0.29;
+
+const styles = {
+  container: {
+    paddingTop({ inputLayoutDimensions }) {
+      // For aligning left label to the center of Text Field
+      if (IS_ANDROID) {
+        return `${inputLayoutDimensions.height * PADDING_TOP_MULTIPLIER_ANDROID}px`;
+      }
+      return `${inputLayoutDimensions.height * PADDING_TOP_MULTIPLIER_IOS}px`;
+    },
+  },
+  padding() {
+    return [0, 3, 0, 0];
+  },
+};
+
+const LabelContainer = styled(View)`
+  padding-top: ${styles.container.paddingTop};
```

Comment:
this can be solved with `Space` no need of one more styled thing here

Reply:
I want to pass exact pixels which are calculated using the layout of the text input and a multiplier which is different for android & ios. Space multiplies it by 8.